sdk/formrecognizer/ai-form-recognizer/karma.conf.js

```diff
@@ -118,13 +118,13 @@ module.exports = function(config) {
     customLaunchers: {
       ChromeHeadlessNoSandbox: {
         base: "ChromeHeadless",
-        flags: ["--no-sandbox", "--disable-web-security"]
-      }
+        flags: ["--no-sandbox", "--disable-web-security"],
+      },
     },
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: false,
+    singleRun: true,
```

Comment:
I agree with this change based on the [docs](https://karma-runner.github.io/6.3/config/configuration-file.html#singlerun) but I am not sure why we have it as false everywhere.

Reply:
I'm not sure either, would be good to look into for MQ as we rework karma/rollup configs.